akka-docs/src/main/paradox/scala/persistence.md

```diff
@@ -1039,10 +1039,36 @@ Java
 ### Eager initialization of persistence plugin
 
 By default, persistence plugins are started on-demand, as they are used. In some case, however, it might be beneficial
-to start a certain plugin eagerly. In order to do that, you should first add the `akka.persistence.Persistence`
+to start a certain plugin eagerly. In order to do that, you should first add `akka.persistence.Persistence`
 under the `akka.extensions` key. Then, specify the IDs of plugins you wish to start automatically under
 `akka.persistence.journal.auto-start-journals` and `akka.persistence.snapshot-store.auto-start-snapshot-stores`.
 
+For example, if you want eager initialization for the leveldb journal plugin and the local snapshot store plugin, your configuration should look like this:  
+
+```
+akka {
+
+  extensions = [
+    akka.persistence.Persistence
+  ]
```

Comment:
I'd write that as `extensions = [akka.persistence.Persistence]` 
or `extensions = ["akka.persistence.Persistence"]`